README.md

```diff
@@ -32,6 +32,7 @@
   - `ost-exercise-choice` : Used for a group of exercises which will be used in the i-reading, but will be pre-processed by Tutor first. Only occurs on `practice-problems` right now. 
   - `ost-reading-discard` : On `snap-lab` and end of section and chapter items, including `key-equations` `key-terms` `glossary` `summary`  `practice-concepts` `chapter-review` (and its variants( `test prep` (and its variants)
   - `ost-assignable`: Only on `snap-lab`s and `chapter-review performance`
+  -  `ost-reading-unwrap`: Used to set apart a pool of either Chapter Review questions or Test Prep questions
```

Comment:
Don't know if we care about this at all. Is this the end of section stuff? We are supposed to just discard CNX's and replaced with our own spaced practice.